src/main/java/com/github/hcsp/inheritance/animal.java

```diff
@@ -0,0 +1,8 @@
+package com.github.hcsp.inheritance;
+
+public class animal {
+    public String name;
```

Comment:
```suggestion
    private String name;
```

面向对象三大原则中的隐藏原则：不要暴露public的属性。

Reply:
难道除了`private`/`public`就没有其他选择了么。。。`protected`去哪儿了？